www/src/components/users/MagicLogin.tsx

```diff
@@ -570,6 +570,7 @@ export function Signup() {
       history.navigate('/')
     }
   }, [history])
+  // we should probably move hubspot to loaded similarly to with the other analytic tools and setup with cookiebot
   useScript({ src: 'https://js.hs-scripts.com/22363579.js' })
```

Comment:
It was originally requested that we only needed hubspot on the login page, so I only put it here to reduce the number of places we'd need to show its consent banner, but now that we're adding a global consent banner, may as well just put it up with the rest as a regular script tag marked with `type="text/plain" data-cookieconsent="statistics"`. Also will want to add some variation on [this](https://github.com/pluralsh/webflow-helpers/blob/main/standalone/tracking-mgmt.js) to disable/re-enable hubspot when prefs change.